### PR TITLE
Start onboarding time entry after authentication

### LINKED ISSUE #7345: Start a new time entry after signup or login
## 🎯 What needs to be done?
We should start a time entry for some specific users (described below) as a part of onboarding.

## 📝 Detailed explanation
The new improved onboarding flow guides the user through the app by creating the first time entry for them auto-magically.

## 🎨 Graphical resources
https://toggl.invisionapp.com/share/2YX4NEV7WF5#/screens/416141745

## 🛠 Technical remarks
**For whom should the onboarding time entry be created?**
* For all new users who sign up and haven't seen the onboarding (either the old one or the new one)
* For users who log in and:
  * Haven't seen the onboarding
  * Have no time entries after logging in (no running entry & no stopped entries)

**What should the time entry look like?**
The time entry should have the following description: `Getting started with Toggl app`

**Still confused?**
Check [this flowchart](https://miro.com/app/board/o9J_kwnl_Z0=/?moveToWidget=3074457347628738556&cot=6) of the new onboarding.

## 🏆 Outcome
- [x] The onboarding time entry is started for users who sign up
- [x] The onboarding time entry is started for users who log in and have no time entries
- [x] The onboarding time entry **is not** started for users who have already seen either old onboarding or the new one

### PR DESCRIPTION
**Pull request by arturs105**
## What's this?
This PR starts the very first time entry for users as a part of onboarding.

### Relationships

Closes #7345 

## :squid: Permissions
🦑

### CODE REVIEW

File: Toggl.Droid/Extensions/OnboardingExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+using System.Reactive.Linq;
+using Toggl.Shared;
+using Toggl.Shared.Extensions;
+using Toggl.Storage.Extensions;
+using Toggl.Storage.Onboarding;
+using Toggl.Storage.Settings;
+
+namespace Toggl.Droid.Extensions
+{
+    public static class OnboardingExtensions
```
Comment (heytherewill) [inline Toggl.Droid/Extensions/OnboardingExtensions.cs:15]: I though we agreed on not going this route. I'd rather just have the tooltips included in the actual layout and simply show/hide them instead.

File: Toggl.iOS/Views/Onboarding/MostUsedPage.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+using Foundation;
+using ObjCRuntime;
+using System;
+using Toggl.iOS.Extensions;
+using UIKit;
+
+namespace Toggl.iOS
+{
+    public sealed partial class MostUsedPage : UIView
```
Comment (heytherewill) [inline Toggl.iOS/Views/Onboarding/MostUsedPage.cs:9]: What even is this? 🤔

**Comment by heytherewill** [comment]
Also, the commit history is really messy. It'd be good if we can get a better commit history for PRs that touch this many files

**Comment by arturs105** [comment]
@heytherewill This PR was still WIP, when you reviewed it and there were some traces from a merge conflict (or some other git debacle) gone bad. I'll make a new PR with just the correct changes and a cleaner history.

**Comment by arturs105** [comment]
This is nice. But [this](https://github.com/toggl/mobileapp/pull/7467) one's even better.

**Pull request closed without merging**